src/test/java/calculator/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,68 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringAddCalculatorTest {
+
+    @Test
+    @DisplayName("null 또는 빈문자가 입력되는 경우 0을 반환하는지 확인한다.")
+    public void splitAndSum_null_또는_빈문자() {
+        int result = StringAddCalculator.splitAndSum(null);
+        assertThat(result).isEqualTo(0);
```

Comment:
```suggestion
        assertThat(result).isZero();
```
0과 비교 시 isZero 메소드 사용을 추천드립니다!